apertium-uzb-kaa.uzb-kaa.dix

```diff
@@ -19553,6 +19553,5526 @@
+<e>       <p><l>Grebennikova<s n="np"/><s n="cog"/></l>    <r>Grebennikova<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Green<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Green<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Gregoresku<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Gregoresku<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Grgich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Grgich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Grigorenko<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Grigorenko<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Grigoresku<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Grigoresku<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Grigoropolos<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Grigoropolos<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Grigoryev<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Grigoryev<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Grigoryeva<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Grigoryeva<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Grikorenko<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Grikorenko<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Grimason<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Grimason<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Grinenko<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Grinenko<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Grishchenko<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Grishchenko<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Gritsenko<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Gritsenko<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Grivichich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Grivichich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Grlich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Grlich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Gromenko<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Gromenko<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Gromov<s n="np"/><s n="cog"/></l>    <r>Gromov<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Gromova<s n="np"/><s n="cog"/></l>    <r>Gromova<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Groothousen<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Groothousen<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Grossi<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Grossi<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Grozdanovskiy<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Grozdanovskiy<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Grubesich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Grubesich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Grubisich<s n="np"/><s n="cog"/><s 
```

Comment:
Shouldn't this be `f`?  There are a number of `cog` and `pat` that appear to be gendered, but without gender tags.

Reply:
I did it like in kaz-tat.dix The tag NP-COG-OB in kaz.lexc  was as "np-cog" in dix. And so NP-PAT-VICH was as np-pat.

Reply:
Oh, you're right, in the bilingual dictionary you don't want gender for ones that can be both.